tools/clang/include/clang/SPIRV/EmitVisitor.h

```diff
@@ -39,11 +40,11 @@ struct SpirvLayoutRuleDenseMapInfo {
 
 class EmitTypeHandler {
 public:
-  EmitTypeHandler(SpirvContext &c, SpirvModule &m,
-                  std::vector<uint32_t> *decVec,
-                  std::vector<uint32_t> *typesVec)
-      : context(c), module(m), annotationsBinary(decVec),
-        typeConstantBinary(typesVec) {
+  EmitTypeHandler(SpirvContext &c, std::vector<uint32_t> *decVec,
+                  std::vector<uint32_t> *typesVec,
+                  std::function<uint32_t()> takeNextIdFn)
```

Comment:
Minor: but we can do `const &` to avoid copying the function object twice.

Reply:
Done